.gitignore

```diff
@@ -29,5 +29,9 @@ bin/
 include/
 lib/
 distribute*
+<<<<<<< HEAD
 share/
 .DS_store
+=======
+share/
+>>>>>>> 6ce90a1a7bfb0383cddc5ca47efab0769f97a51d
```

Comment:
Please revert all changes to that file.

Reply:
Done. Apologize was dealing with some rebase goofyness I was working my way through.

Reply:
No worries, my bad. I was just about to publish my review when those came up. I probably should have waited a few moments more.